src/components/PreJoinScreens/DeviceSelectionScreen/DeviceSelectionScreen.test.tsx

```diff
@@ -35,8 +36,8 @@ describe('the DeviceSelectionScreen component', () => {
 
     const wrapper = shallow(<DeviceSelectionScreen name="test name" roomName="test room name" setStep={() => {}} />);
 
-    it('should disable the Join Now button', () => {
-      expect(wrapper.find({ children: 'Join Now' }).prop('disabled')).toBe(true);
+    it('should show the loading screen', () => {
+      expect(wrapper.find(CircularProgress).prop('variant')).toBe('indeterminate');
```

Comment:
Nit - Here I don't think we need to look at the `variant` prop. It's hardcoded and won't change. We just need to know if the spinner is there or not, so I think we can do something like this (same with line 84):

```suggestion
      expect(wrapper.find(CircularProgress).exists()).toBe(true);
```